backend/api/schema.py

```diff
@@ -7,14 +7,14 @@
 from voting.mutations import VotesMutations
 
 
-class Query(UsersQuery, ConferenceQuery, graphene.ObjectType):
+@strawberry.type
```

Comment:
does it work without the decorator? just curious :D

Reply:
Without decorator :

![image](https://user-images.githubusercontent.com/3382153/61976222-7cf33600-afeb-11e9-8a49-e1f0727912d4.png)


It seems like only the first base class is read, ignoring the remaining stuff (the first `type` class?)